kube-burner/kubelet-density/kubelet-density.yaml

```diff
@@ -0,0 +1,35 @@
+---
+global:
+  gc: true
+  indexerConfig:
+    esServers: ["https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443"]
+    insecureSkipVerify: true
+    defaultIndex: "ripsaw-kube-burner"
+    type: opensearch
```

Comment:
There's a new Configuration for the indexer, I'm using v1.10.0 kube-burner version
https://kube-burner.github.io/kube-burner/latest/reference/configuration/#reference


```
---
global:
  gc: true
  measurements:
    - name: podLatency

indexers:
 - type: opensearch 
   esServers: ["https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443""]
   insecureSkipVerify: true
   defaultIndex: "ripsaw-kube-burner"
 ```